src/interface/interface.ts

```diff
@@ -242,6 +242,7 @@ export interface Round2Info {
 export enum Round2Status {
   ProofNotRegistered = "ProofNotRegistered",
   ProofRegistered = "ProofRegistered",
+  Archived = "Archived",
```

Comment:
This change need discuss, why we need this one. 
Archived should not be a status combined with ProofRegistered or not status.
It is either a new status (which most likely an intern usage) if it is necessary, or it should not be added in round 2 info. It should be something in the archive related metadata, like a latest archived task id so all task which id is less than that one should be considered as archived.

This one is not good as normally client's will use this status to judge whehther they can do verify for the proof tasks so generally I don't think we should add this. We can discuss the purpose of it.